services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+from typing import Optional
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list | None = None,
+        actors_ids: list | None = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids is not None:
+        movies = movies.filter(genres__id__in=genres_ids).distinct()
+
+    if actors_ids is not None:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()
```

Comment:
just `if actors_ids` is enough because python checks if it isn't None, False, empty string or list, etc